### PR TITLE
{Release} Upgrade to Azure CLI 2.62.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

**Comment by yonzhan** [comment]
Release

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,118 @@
 Release History
 ===============
 
+2.62.0
+++++++
+
+**ACR**
+
+* `az acr build` and `az acr task`: Highlight the difference between arguments in the description (#28941)
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:11]:
```suggestion
* `az acr build/task`: Highlight the difference between arguments in the description (#28941)
```

Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:11]:
```suggestion
* `az acr build/task`: Highlight the difference between arguments in the description (#28941)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,118 @@
...
+**Backup**
+
+* `az backup restore restore-disks/restore-azurewl/restore-azurefiles/restore-azurefileshare`: Add `--tenant-id` for cross-tenant MUA protection, otherwise allows restore when protected by a resource guard (#28950)
+* `az backup vault encryption update`: Add support for MUA for CMK operations (#29213)
+
+**CDN**
+
+* Fix #28721: `az afd origin-group create`: Add parameter to support session affinity state (#28995)
+* Fix #28824: `az cdn profile`: Add default location (#28996)
+* Fix 28733: `az cdn endpoint update`: Cannot update an existing endpoint (#29110)
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:46]:
```suggestion
* Fix #28733: `az cdn endpoint update`: Cannot update an existing endpoint (#29110)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,118 @@
...
+
+* `az aks create/update`: Update AMW resource.location to only accept alphanumeric characters (#29025)
+* `az aks create/update`: Add Azure Managed Grafana support for managed prometheus addon in usnat (#29098)
+* `az aks create/update`: Ephemeral disk additional support for Azure Container Storage (#29230)
+* `az aks create/update`: Add and fix existing validations for ephemeral disk support for Azure Container Storage (#29274)
+
+**App Config**
+
+* Update appconfig help messages (#29151)
+* `appconfig kv import`: Update the help message for ignore-match (#29200)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:23]:
```suggestion
* `az appconfig kv import`: Update the help message for ignore-match (#29200)
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,118 @@
...
+**MySQL**
+
+* `az mysql flexible-server update`: Support replica resource group different with primary server (#29160)
+* `az mysql flexible-server detach-vnet`: New command to support converting a vnet azure mysql server to a non-vnet server (#29231)
+* `az mysql flexible-server deploy/update`: Add warning message for HA enabled and input validator (#29229)
+
+**NetAppFiles**
+
+* Coolnessperiod minimum fix (#29050)
+
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:85]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,118 @@
...
+**AKS**
+
+* `az aks create/update`: Update AMW resource.location to only accept alphanumeric characters (#29025)
+* `az aks create/update`: Add Azure Managed Grafana support for managed prometheus addon in usnat (#29098)
+* `az aks create/update`: Ephemeral disk additional support for Azure Container Storage (#29230)
+* `az aks create/update`: Add and fix existing validations for ephemeral disk support for Azure Container Storage (#29274)
+
+**App Config**
+
+* Update appconfig help messages (#29151)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:22]:
```suggestion
* `az appconfig kv import`: Update help message for App Configuration store (#29151)
```

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,118 @@
...
+* `az postgres flexible-server migration create`: Fix bugs while passing in MigrationRuntimeResourceId to Migration Parameters, now need to be provided within properties json file (#29106)
+* `az postgres flexible-server firewall-rule create`: Correct firewall rule name and ip range validators (#29224)
+* `az postgres flexible-server update`: Add argument to enable or disable public-access (#29228)
+* `az postgres flexible-server create`: Add argument `--create-default-database` to support disabling default database creation (#29227)
+* `az postgres flexible-server upgrade`: Unblock MVU for Burstable from CLI (#29145)
+* `az postgres flexible-server update`: Correct setting `--maintenance-window` to be disabled (#29249)
+
+**SignalR**
+
+* Remove the empty location string when updating resource (#29242)
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:107]:
```suggestion
* `az signalr upstream update`: Remove the empty location string when updating resource (#29242)
```

File: src/azure-cli/HISTORY.rst (hunk 3 above)
Comment (yanzhudd) [inline src/azure-cli/HISTORY.rst:23]:
```suggestion
* `az appconfig kv import`: Update the help message for ignore-match (#29200)
```

File: src/azure-cli/HISTORY.rst (hunk 5 above)
Comment (yanzhudd) [inline src/azure-cli/HISTORY.rst:22]:
```suggestion
* `az appconfig`: Update appconfig help messages (#29151)
```

File: src/azure-cli/HISTORY.rst [hunk 7]
```diff
@@ -3,6 +3,118 @@
...
+* Fix 28733: `az cdn endpoint update`: Cannot update an existing endpoint (#29110)
+
+**Compute**
+
+* `az vmss update`: Add new parameter `--zones` to support setting vmss zonal expansion (#29035)
+* `az vm install-patches`: Add new parameter `--max-patch-publish-date` to support setting max patch publish date (#29045)
+* `az vmss reimage`: Add new parameter `--force-update-os-disk-for-ephemeral` to support updating the base OS disk (#29085)
+* `az vmss create/update`: Add new parameter `--scheduled-event-additional-publishing-target-event-grid-and-resource-graph` to support setting scheduled event policy (#29122)
+* `az vmss create/update`: Add new parameter `--enable-user-reboot-scheduled-events` to support setting scheduled event policy (#29122)
+* `az vmss create/update`: Add new parameter `--enable-user-redeploy-scheduled-events` to support setting scheduled event policy (#29122)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:55]:
```suggestion
* `az vmss create/update`: Add new parameter `--enable-user-reboot-scheduled-events` to support setting policy for rebooting scheduled event (#29122)
* `az vmss create/update`: Add new parameter `--enable-user-redeploy-scheduled-events` to support setting policy for redeploying scheduled event (#29122)
```

File: src/azure-cli/HISTORY.rst [hunk 8]
```diff
@@ -3,6 +3,114 @@
...
+**App Service**
+
+* `az webapp deploy`: Disable deploymentstatus API for deployments to app slots (#29209)
+* `az webapp deployment source config-zip`: Disable deploymentstatus API for deployments to app slots (#29209)
+* Fix #29041: `az webapp config access-restriction add`: Fix edge case validating service tags (#29048)
+* `az functionapp create`: Do not throw exception if endOfLifeDate not passed from stacks API (#28974)
+
+**ARM**
+
+* `az stack mg create`: MG stacks are now automatically validated before creation (#29092)
```
Comment (yanzhudd) [inline src/azure-cli/HISTORY.rst:34]:
```suggestion
* `az stack mg create`: Validate MG stacks automatically before creation (#29092)
```

File: src/azure-cli-core/HISTORY.rst [hunk 9]
```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+2.62.0
+++++++
```
Comment (jiasli) [inline src/azure-cli-core/HISTORY.rst:7]:
```suggestion
++++++
Fix #28997: Fix error "User cancelled the Accounts Control Operation" when logging in with WAM as Administrator
```

Comment (jiasli) [inline src/azure-cli-core/HISTORY.rst:7]:
```suggestion
++++++
Fix #28997: Fix error "User cancelled the Accounts Control Operation" when logging in with WAM as Administrator (#29088)
```

File: src/azure-cli/HISTORY.rst [hunk 10]
```diff
@@ -3,6 +3,118 @@
...
+
+* Update appconfig help messages (#29151)
+* `appconfig kv import`: Update the help message for ignore-match (#29200)
+
+**App Service**
+
+* `az webapp deploy`: Disable deploymentstatus API for deployments to app slots (#29209)
+* `az webapp deployment source config-zip`: Disable deploymentstatus API for deployments to app slots (#29209)
+* Fix #29041: `az webapp config access-restriction add`: Fix edge case validating service tags (#29048)
+* `az functionapp create`: Do not throw exception if endOfLifeDate not passed from stacks API (#28974)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:30]:
```suggestion
* `az functionapp create`: Avoid throwing exception when endOfLifeDate not passed from stacks API (#28974)
```

File: src/azure-cli/HISTORY.rst [hunk 11]
```diff
@@ -3,6 +3,114 @@
...
+**Backup**
+
+* `az backup restore restore-disks/restore-azurewl/restore-azurefiles/restore-azurefileshare`: Add `--tenant-id` for cross-tenant MUA protection, otherwise allows restore when protected by a resource guard (#28950)
+* `az backup vault encryption update`: Add support for MUA for CMK operations (#29213)
+
+**CDN**
+
+* Fix #28721: `az afd origin-group create`: Add parameter to support session affinity state (#28995)
+* Fix #28824: `az cdn profile`: Add default location (#28996)
+* Fix #28733: `az cdn endpoint update`: Cannot update an existing endpoint (#29110)
```
Comment (necusjz) [inline src/azure-cli/HISTORY.rst:46]:
```suggestion
* Fix #28733: `az cdn endpoint update`: Add support to update an existing endpoint (#29110)
```

File: src/azure-cli-core/HISTORY.rst [hunk 12]
```diff
@@ -3,6 +3,13 @@
...
+Fix #28997: Fix error "User cancelled the Accounts Control Operation" when logging in with WAM as Administrator (#29088)
```
Comment (jiasli) [inline src/azure-cli-core/HISTORY.rst:8]:
```suggestion
* Fix #28997: Fix error "User cancelled the Accounts Control Operation" when logging in with WAM as Administrator (#29088)
```

**Review by jiasli — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Review by yanzhudd — APPROVED** [review]

**Review by bebound — APPROVED** [review]

**Review by calvinhzy — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**